validate/validate.go

```diff
@@ -221,20 +229,30 @@ func isSvnHigherOrEqual(quoteSvn []byte, optionSvn []byte) bool {
 }
 
 func minVersionCheck(quote *pb.QuoteV4, opts *Options) error {
+	logger.V(1).Info("Setting the minimum_qe_svn parameter value to ", opts.HeaderOptions.MinimumQeSvn)
+	logger.V(1).Info("Setting the minimum_pce_svn parameter value to ", opts.HeaderOptions.MinimumPceSvn)
+	logger.V(1).Info("Setting the minimum_tee_tcb_svn parameter value to ", opts.TdQuoteBodyOptions.MinimumTeeTcbSvn)
+
 	if !isSvnHigherOrEqual(quote.GetTdQuoteBody().GetTeeTcbSvn(), opts.TdQuoteBodyOptions.MinimumTeeTcbSvn) {
 		return fmt.Errorf("TEE TCB security-version number %d is less than the required minimum %d",
 			quote.GetTdQuoteBody().GetTeeTcbSvn(), opts.TdQuoteBodyOptions.MinimumTeeTcbSvn)
 	}
+	logger.V(2).Infof("TEE TCB security-version number(%v) is not less than minimum_tee_tcb_svn value(%v)", quote.GetTdQuoteBody().GetTeeTcbSvn(), opts.TdQuoteBodyOptions.MinimumTeeTcbSvn)
```

Comment:
How about
TEE TCB security-version number(%v) is **greater** than minimum_tee_tcb_svn value(%v)

Reply:
Done